shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -0,0 +1,44 @@
+- 8:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    os_variant = rhel8
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 2048
+    nic_hotplug:
+        modprobe_module =
+    block_hotplug:
+        modprobe_module =
+    no unattended_install..floppy_ks
+    unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
+        cdrom_unattended = images/${os_variant}-${vm_arch_name}/ks.iso
+        syslog_server_proto = udp
+    unattended_install, svirt_install:
+        kernel = images/${os_variant}-${vm_arch_name}/vmlinuz
+        initrd = images/${os_variant}-${vm_arch_name}/initrd.img
+        # ARCH dependent things
+        aarch64:
+            grub_file = /boot/grub/grub.conf
```

Comment:
For aarch64, no this file, but `/boot/efi/EFI/redhat/grub.cfg`

Reply:
@yanan-fu are you sure? Is there any documentation about it? There is the same path for `aarch64` in case of `RHEL.7.devel`  https://github.com/avocado-framework/avocado-vt/blob/339cfd2eba2dbb88c7a18fdf83ba6ea00c219876/shared/cfg/guest-os/Linux/RHEL/7.devel/aarch64.cfg#L2

So this line needs correction as well?

Reply:
Yes, i checked with a latest rhel8 host for arm:
```
[root@hp-moonshot-03-c07 ~]# uname -r
4.18.0-67.el8.aarch64
[root@hp-moonshot-03-c07 ~]# ls /boot/grub2/
grubenv
[root@hp-moonshot-03-c07 ~]# ls /boot/efi/EFI/redhat/grub.cfg 
/boot/efi/EFI/redhat/grub.cfg
```

Reply:
@rdkdd  For aarch64 vm,  result is same with above.
From this document:  https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html-single/8.0_beta_release_notes/index#infrastructure_services_3
 it use `/etc/grub2.cfg`, and it is a soft link: 
aarch64:
```
ll /etc/grub2-efi.cfg 
lrwxrwxrwx. 1 root root 31 Oct 19 17:42 /etc/grub2-efi.cfg -> ../boot/efi/EFI/redhat/grub.cfg
```
x86:
```
ll /etc/grub2.cfg 
lrwxrwxrwx. 1 root root 22 Dec 19 13:02 /etc/grub2.cfg -> ../boot/grub2/grub.cfg
```

For `RHEL.7.devel`, i am not very sure what it used for since i never use it. What i know is RHEL-ALT, aarch64 use `/boot/efi/EFI/redhat/grub.cfg` too. @ldoktor  do you know it ?